src/pages/blog/the-perfect-react-bootcamp-student.md

```diff
@@ -0,0 +1,92 @@
+order: 15
+---
+
+We’ve been teaching React since 2016 and in that time we’ve often been asked what makes the perfect ReactJS Academy student. We’ve been asked what base knowledge you need, especially. So today we’re going to answer a few FAQs to make sure you get the most out of React Bootcamp - and we’ll have some snippets from past students along the way!
+
+![image alt text](https://firebasestorage.googleapis.com/v0/b/reactjsacademy-react.appspot.com/o/blog%20post%20images%2FperfectReactStudent%2Fimage_1.jpg?alt=media&)
+
+## What do ReactJS Academy students learn?
+
+Just to recap what the React Bootcamp is, it’s a week-long intensive React training course. We introduce students to React and get them thinking in a React way. Then we mentor students rapidly, taking a deep dive into the React ecosystem to cover:
+
+- Modern JS
+
+- Forms
+
+- Authentication
+
+- Styling
+
+- Redux
+
+- Testing principles and testing in React
+
+- Functional programming
+
+- Advanced React patterns
+
+- GraphQL
+
+Obviously, this kind of React training and GraphQL course can be pretty intense. It isn’t for the faint-hearted, but we make sure classes are small enough so that students get the most out of their face-to-face React and GraphQL training.
+
+**To get the most out of the Bootcamp, you’ve got to be at the right stage in your career and at a certain level of experience.**
+
+![image alt text](https://firebasestorage.googleapis.com/v0/b/reactjsacademy-react.appspot.com/o/blog%20post%20images%2FperfectReactStudent%2Fimage_2.jpg?alt=media&)
+
+## How much experience do I need to learn React?
+
+As I said above, the React Bootcamp is intense and rapid. **We’re not here to introduce you to coding, we’re here to supplement your knowledge and introduce you to a new technology.** So the students who get the most out of our React training course are experienced developers.
+
+If you’re familiar with JavaScript but want to learn a new, efficient way to use it - and fast! - you’re ready for the React Bootcamp.
+
+Although the course is intense, it’s worth it. One former student, Polina Stoyanova from tray.io, said she found the course "fantastic".
+
+> "It was really tiring, but it was all worth it. [...] Overall, it taught me many things I didn’t know that I didn’t know about React! So I would definitely recommend putting the time and effort in."
+
+<video youtube-id="6hmKu1-vW-8" ></video>
+
+On another note, if you’re a manager the Bootcamp is perfect for levelling out your team. Our course will get your devs up to speed and enhance the knowledge of your best employees. Everyone will be on the same wavelength afterwards, and that team cohesion will pay dividends for you going forward. Find out more about how team training can level your team’s skills [on our blog](https://reactjs.academy/blog/5-reasons-why-advanced-dev-training-will-keep-your-company-tech-relevant/).
+
+## Who will get the most out of the React Bootcamp?
```

Comment:
@ric9176 @shakersinthedark I found out that adding an anchor in the titles helps for link building. Example, you can have at the bottom a "wrap up / recap" that includes a link to those titles. Then when you run the auto import of the post in Medium we get a lot of link to our website. Example, the conclusion at the bottom (https://reactjs.academy/blog/react-forms-controlled-and-uncontrolled-components/) has a link to the section "Form library" https://reactjs.academy/blog/react-forms-controlled-and-uncontrolled-components/#form-library. And it's also the same in Medium https://medium.com/leanjs/react-forms-controlled-uncontrolled-components-7842a7284dfa, though the Medium links take the user to the section in our blog